backend/hitas/calculations/max_prices/max_price.py

```diff
@@ -216,8 +216,16 @@ def calculate_max_price(
             "official_name": apartment.housing_company.official_name,
             "archive_id": apartment.housing_company.id,
             "property_manager": {
-                "name": apartment.housing_company.property_manager.name,
-                "street_address": apartment.housing_company.property_manager.street_address,
+                "name": safe_attrgetter(
+                    apartment.housing_company,
+                    "property_manager.name",
+                    default="",
+                ),
+                "street_address": safe_attrgetter(
+                    apartment.housing_company,
+                    "property_manager.street_address",
+                    default="",
+                ),
```

Comment:
I wonder it would be clearer to have `property_manager` be `None` if it's not set instead of returning a blank string 🤔

Reply:
It's simpler this way since code using the property manager info does not need to handle Nones explicitly, e.g. [confirmed_maximum_price.jinja:150](https://github.com/City-of-Helsinki/hitas/blob/8b78560ce01d5c1fd9e06027026977089cde7b8f/backend/hitas/templates/confirmed_maximum_price.jinja#L150).